### PR TITLE
uploading  /MI-10-MEAN/Pratiksha/form/form.html/form.css

### PR DESCRIPTION
**Pull request by pratikshaghule**

### CODE REVIEW

**Review by omerana14 — CHANGES_REQUESTED** [review]
Leave blank space in last line of the program

**Review by MKadeer17 — APPROVED** [review]
Good

File: Pratiksha/html/Form/form.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="form.css">
+</head>
+<body>
+   <div class="main">
+       <h1 >MI-1o Intern Assessment</h1>
```
Comment (VIP5614) [inline Pratiksha/html/Form/form.html]: all h1 tag inside div tag

Comment (VIP5614) [inline Pratiksha/html/Form/form.html]: MI-1o Intern Assessment, Personal Details and Academic Details in sab ko aap div tag ke undar h2 aur h3 ke under rakhiye ye proper way ho ga

File: Pratiksha/html/Form/form.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (VIP5614) [inline Pratiksha/html/Form/form.html:3]: aap proper structure kyo nahi deti hai 
<html>
      <head>
      </head>
</html> 
aay se hoga na

File: Pratiksha/html/Form/form.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="form.css">
+</head>
+<body>
```
Comment (VIP5614) [inline Pratiksha/html/Form/form.html]: 1 same problem

**Review by VIP5614 — APPROVED** [review]

File: Pratiksha/html/Form/form.css [hunk 4]
```diff
@@ -0,0 +1,10 @@
+.blue{
+    padding: 5px;
+    background-color: blue;
+    text-align: right;
+    font-style: italic;
```
Comment (VIP5614) [inline Pratiksha/html/Form/form.css:5]: add border-radius ok

**Review by VIP5614 — APPROVED** [review]

File: Pratiksha/html/Form/form.html [hunk 5]
```diff
@@ -7,32 +7,31 @@
     <title>Document</title>
     <link rel="stylesheet" href="form.css">
 </head>
```
Comment (VIP5614) [inline Pratiksha/html/Form/form.html:9]: head ka kon sahi kare ga Miss

**Review by VIP5614 — APPROVED** [review]

File: Pratiksha/html/Form/form.html (hunk 1 above)
Comment (pratikshaghule) [reply Pratiksha/html/Form/form.html]: ok

**Review by omerana14 — APPROVED** [review]

**Review by MKadeer17 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**